services/nutanix-ai/metadata.yaml

```diff
@@ -0,0 +1,21 @@
+displayName: Nutanix AI
+description: NAI core components.
```

Comment:
please add dependencies to show the list of dependent apps from catalog

Reply:
updated